libraries/libopaevfio/opaevfiotest.c

```
@@ -57,7 +58,8 @@ void print_dfhs(struct opae_vfio *v)
 	       mmio->fme_version, mmio->afu_id_low,
 	       mmio->afu_id_high, mmio->next_afu);
 
-	opae_vfio_region_get(v, 2, (uint8_t **)&mmio, NULL);
+	if (opae_vfio_region_get(v, 2, (uint8_t **)&mmio, NULL))
+		return;
```

Print something when returning early so that this isn't a silent failure.